### PR TITLE
Add EFK for logging

### PR DESCRIPTION
**Pull request by niqdev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,11 +200,40 @@ For a working example on DigitalOcean using [`external-dns`](https://github.com/
     [open|xdg-open] http://localhost:8004
     ```
 
+*EFK stack for logging*
+
+* [`elasticsearch`](https://github.com/elastic/helm-charts/tree/master/elasticsearch) is a distributed, RESTful search and analytics engine and it's is used for log storage
+    ```bash
+    kubectl port-forward service/elasticsearch-master 9200:9200 -n observe
+    [open|xdg-open] http://localhost:9200
```
Comment (conorfennell) [inline README.md]: remove the open command, having the port forward is enough

File: README.md [hunk 2]
```diff
@@ -200,11 +200,40 @@ For a working example on DigitalOcean using [`external-dns`](https://github.com/
...
+    ```
+
+* [`cerebro`](https://github.com/helm/charts/tree/master/stable/cerebro) is an Elasticsearch web admin tool
+    ```bash
+    kubectl port-forward service/cerebro 9000:80 -n observe
+    [open|xdg-open] http://localhost:9000
```
Comment (conorfennell) [inline README.md]: remove the open command, having the port forward is enough

File: README.md [hunk 3]
```diff
@@ -200,11 +200,40 @@ For a working example on DigitalOcean using [`external-dns`](https://github.com/
...
+    ```
+
+* [`kibana`](https://github.com/elastic/helm-charts/tree/master/kibana) lets visualize and query the log data stored in an Elasticsearch index
```
Comment (conorfennell) [inline README.md]:
```suggestion
* [`kibana`](https://github.com/elastic/helm-charts/tree/master/kibana) visualize and query the log data stored in an Elasticsearch index
```

File: README.md [hunk 4]
```diff
@@ -200,11 +200,40 @@ For a working example on DigitalOcean using [`external-dns`](https://github.com/
...
+    ```bash
+    kubectl port-forward service/kibana-kibana 9001:5601 -n observe
+    [open|xdg-open] http://localhost:9001
```
Comment (conorfennell) [inline README.md]: remove the open command, having the port forward is enough

File: applications/templates/observe/elasticsearch.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  name: elasticsearch
+spec:
+  project: observe
+  source:
+    repoURL: https://github.com/elastic/helm-charts.git
+    path: elasticsearch
+    helm:
+      # values from the ui can't be overriden when using file
+      #valueFiles:
+      #  - examples/minikube/values.yaml
```
Comment (conorfennell) [inline applications/templates/observe/elasticsearch.yaml]: remove the dead code and comment

File: applications/templates/observe/fluent-bit.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+          value: "false"
+        - name: serviceAccount.create
+          value: "false"
+        - name: backend.type
+          value: es
+        - name: backend.es.host
+          value: elasticsearch-master.observe
+        - name: backend.es.port
+          value: "9200"
+          #parsers.conf: |
```
Comment (conorfennell) [inline applications/templates/observe/fluent-bit.yaml]: remove the dead code or have a sample parser that is used, this doesn't work if uncommented

**Review by conorfennell — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (niqdev) [reply README.md]: I have the same also above, do you think that I should remove all of them?

Comment (conorfennell) [reply README.md]: Yeah, I would 👍

File: applications/templates/observe/elasticsearch.yaml (hunk 5 above)
Comment (niqdev) [reply applications/templates/observe/elasticsearch.yaml]: I'll reference it only then, cos I want to say from where I took those values
```
# see https://github.com/elastic/helm-charts/blob/master/elasticsearch/examples/minikube/values.yaml
```

File: README.md (hunk 1 above)
Comment (niqdev) [reply README.md]: done

**Pull request merged**